dhall/tests/Dhall/Test/QuickCheck.hs

```diff
@@ -385,25 +387,33 @@ isSameAsSelf expression =
 
 tests :: TestTree
 tests =
-    Test.Tasty.QuickCheck.testProperties
+    testProperties'
         "QuickCheck"
         [ ( "Binary serialization should round-trip"
           , Test.QuickCheck.property binaryRoundtrip
+          , QuickCheckTests 100
           )
         , ( "everything well-typed should normalize"
-          , Test.QuickCheck.property
-              (Test.QuickCheck.withMaxSuccess 10000 everythingWellTypedNormalizes)
+          , Test.QuickCheck.property everythingWellTypedNormalizes
+          , QuickCheckTests 100000
```

Comment:
I've also increased the number of tests here by 10x. That means we should probably hold off on merging this until https://github.com/dhall-lang/dhall-haskell/pull/1194 has been merged.